### DISCUSSION TITLE
超基础关于插件使用的问题，是不是我理解错了

### DISCUSSION THREAD

**Discussion by getSilver** [nonebot/discussions#417] [问答]
适配器连接平台，Nonebot的插件给平台机器人提供扩展功能。这里的插件是不是也得做相应的适配？比如我用的TG适配器，那一些插件就得适配TG？但从OneBot 文档中得知只要按照OneBot协议就都可以通用？

目前情况：
成功创建机器人，用Telegram适配器在TG中成功 /echo hello world。走到这里应该是没问题了吧。
可是当我用插件拓展功能时，几乎没有一个插件成功相应过机器人的指令除去 /echo hello world 。
一个群广播插件都不能成功运行。Succeeded to load plugin "nonebot_plugin_broadcast"正常加载

**Comment by yanyongyu** [comment]
插件如果限制了执行平台，那么就不能在除插件适配的平台外的适配器执行，你可以在 https://registry.nonebot.dev/ 查看插件支持信息

**Reply by getSilver** [reply]
限制执行平台，那元数据包含了执行平台说明该插件可以在包含的平台运行了？
supported_adapters | ["~telegram","~qqguild","~onebot.v11","~kaiheila","~console","~onebot.v12","~feishu"]

**Reply by yanyongyu** [reply]
如果包含了平台那就是可以执行的平台
